torch/_torch_docs.py

```diff
@@ -1655,14 +1655,8 @@ def merge_dicts(*dicts):
 a resulting tensor:
 
 .. math::
-    y_i = \begin{cases}
-        \text{min} & \text{if } x_i < \text{min} \\
-        x_i & \text{if } \text{min} \leq x_i \leq \text{max} \\
-        \text{max} & \text{if } x_i > \text{max}
-    \end{cases}
+    y_i = \min(\text{max}, \min(\text{min}, x_i))
```

Comment:
min(max, min(min, x)?

This is a good lesson in why param names should never be "min" or "max." Two issues here:

- is this correct? I would have expected min(max(x, min), max)?
- how about using different names for the parameters and the functions? This could be done in one of two ways: the functions can be renamed to minimum and maximum or this can use a let expression ("let min_value and max_value be :attr:\`min\` and :attr:\`max\`, respectively, this computes...").

Reply:
That was a mistake indeed, thanks for spotting it! I've updated this with a "let" expression as you've suggested to redefine the arguments, since on a quick skim it seems like "min" and "max" function names are used more often in the documentation than "minimum" and "maximum". Also, I agree that these argument names aren't great, but changing them would also change function definitions, so I kept the names to what was originally used before.

Reply:
Right. Sorry, didn't mean to suggest you named them.